tests/acceptance/blocking_adapter_test.py

```diff
@@ -476,13 +469,13 @@ def test(self):
         self.assertIsInstance(method, pika.spec.Basic.GetOk)
         self.assertEqual(method.delivery_tag, 1)
         self.assertFalse(method.redelivered)
-        self.assertEqual(method.exchange, _p3_as_bytes(''))
-        self.assertEqual(method.routing_key, q_name)
+        self.assertEqual(method.exchange, as_bytes(''))
+        self.assertEqual(method.routing_key, as_bytes(q_name))
```

Comment:
True, though would it be possible to have it in a double role of sorts, something like

- bytes -> do what it does at the moment
- something with application provided properties -> do what it does at the moment
- text (unicode codepoints) given, no application provided properties -> encode and provide proper content-type allowing magic decoding at consumer end

Just a thought, might as well keep it as is.